fast_llm/layers/transformer/vision_transformer.py

```diff
@@ -0,0 +1,16 @@
+import torch
+
+from fast_llm.engine.config_utils.tensor_space import TensorDim
+from fast_llm.layers.transformer.config import VisionTransformerKwargs
+from fast_llm.layers.transformer.transformer import TransformerLayer
+from fast_llm.tensor import TensorMeta
+
+
+class VisionTransformerLayer(TransformerLayer):
+    _name: str = "Vision transformer layer"
+
+    def _get_meta(self, tensor: torch.Tensor, name: str, kwargs: dict):
```

Comment:
This seems to be the same as in the base class?
If it's the case do we need this class at all?